src/onnxflow/justbuildit/export.py

```diff
@@ -6,6 +6,7 @@
 import copy
 from typing import Union
 import torch
+import numpy as np
```

Comment:
Please rev the onnxflow and mlagility patch version numbers so that we can release a package.

Reply:
Updated both version numbers!